vminfo_parser/vminfo_parser.py

```diff
@@ -65,6 +64,9 @@ def calculate_disk_space_ranges(
         dataFrame[frameHeading] = pd.to_numeric(dataFrame[frameHeading], errors="coerce")
         unit = self.column_headers["unitType"]
 
+        # Normalize the Disk Column to GiB before applying further analysis
+        if unit == "MB":
+            dataFrame[frameHeading] = dataFrame[frameHeading] / 1024
```

Comment:
Is there a reason for doing this here rather than moving this to VMData?  Im not sure whether I wold prefer to move it now, or leave it here until we hand the larger normalization and access challenges.  although im leaning towards the later.

Reply:
The only reason I put this here for now is because it already existed in this function in multiple places. I moved it up top and got rid of the other sections. I am open to moving this if it makes sense. There are 3 calculate functions in analyzer already. We can discuss moving them